pkg/envoy/lds/listener.go

```diff
@@ -51,7 +42,29 @@ func (lb *listenerBuilder) newOutboundListener() (*xds_listener.Listener, error)
 				Name: wellknown.HttpInspector,
 			},
 		},
-	}, nil
+	}
+
+	// Create filter chain for egress if egress is enabled
+	// This filterchain matches any traffic not filtered by allow rules, it will be treated as egress
+	// traffic when enabled
+	if lb.cfg.IsEgressEnabled() {
+		egressFilterChgain, err := buildEgressFilterChain()
+		if err != nil {
+			log.Error().Err(err).Msgf("Error getting filter chain for Egress")
```

Comment:
```suggestion
			log.Error().Err(err).Msgf("Error building filter chain for Egress")
```

Reply:
I'll consider updating this in a subsequent change or if I need to rebase.